annotations/views/date_appellation_views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import viewsets
+from rest_framework import status
+from rest_framework.response import Response
+
+from annotations.models import DateAppellation, DocumentPosition
+from annotations.serializers import DateAppellationSerializer, DocumentPositionSerializer
+
+class DateAppellationViewSet(viewsets.ModelViewSet):
+    queryset = DateAppellation.objects.all()
+    serializer_class = DateAppellationSerializer
+
+    def create(self, request):
+        data = request.data.copy()
+        position = data.pop('position', None)
+        if 'month' in data and data['month'] is None:
+            data.pop('month')
```

Comment:
if you don't need to save the value into another variable I wouldn't use pop, just access it by key. That way your not mutating the original data unnecessarily.